purchases/src/main/kotlin/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -730,9 +734,7 @@ internal class BillingWrapper(
     }
 
     override fun onBillingServiceDisconnected() {
-        mainHandler.post {
-            log(LogIntent.DEBUG, BillingStrings.BILLING_SERVICE_DISCONNECTED.format(billingClient.toString()))
-        }
```

Comment:
I removed wrapping this log to the main thread... Seemed like unneeded overhead.

Reply:
Yeah, not sure why this would be posted to the main thread